modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,133 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any organization.
+Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization (including the Federated ones). It is the recommended initial solution pending future corporate investments in Enterprise specific OAuth 2.0 offerings like
```

Comment:
`It is the recommended initial solution pending future corporate investments in Enterprise specific OAuth 2.0 offerings like` this is too difficult to read, maybe a comma somewhere or a small reformulation 🤔.

Reply:
Done.